proto/dydxprotocol/vault/tx.proto

```diff
@@ -39,6 +39,21 @@ message MsgDepositToVault {
 // MsgDepositToVaultResponse is the Msg/DepositToVault response type.
 message MsgDepositToVaultResponse {}
 
+// MsgWithdrawFromVault withdraws the specified amount of from the vault to the subaccount.
+message MsgWithdrawFromVault  {
+	// The vault to withdraw from.
+	VaultId vault_id = 1;
+
+	// The subaccount to withdraw to.
+	dydxprotocol.subaccounts.SubaccountId subaccount_id = 2;
+
+	// Number of shares to withdraw.
+	uint64 shares = 3;
```

Comment:
the [original spec](https://www.notion.so/dydx/x-vault-caf6cd94c80e448a96a70fe20051aa1c?pvs=4#95a1b5dc21104b18a9b2aff718a32b6e) calls for `quote_quantums` but i think it makes more sense to specify the # of shares. wdyt?

Reply:
offline conclusion: decided to go w specifying USDC

Reply:
offline conclusion: decided to go w specifying shares